### PR TITLE
Change Status to a slice containing Status structs

### PR DESCRIPTION
**Pull request by alde**
The Status struct contains a Time and a Status element, so that a history of
status changes can be viewed.

Solves #13

### CODE REVIEW

File: handler/scheduler.go [hunk 1]
```diff
@@ -156,3 +149,16 @@ func scheduleTask(s *eremeticScheduler, request types.Request) (string, error) {
...
+
+func updateStatusForTask(status *mesos.TaskStatus) {
+	id := status.TaskId.GetValue()
+	log.Debugf("TaskId [%s] status [%s]", id, status.State)
+	task, _ := database.ReadTask(id)
+	newStatus := types.Status{
+		Status: status.State.String(),
+		Time:   time.Now().Unix(),
+	}
+	task.Status = append([]types.Status{newStatus}, task.Status...)
```
Comment (keis) [inline handler/scheduler.go]: hmm. so we're doing a prepend? why?

File: handler/scheduler.go [hunk 2]
```diff
@@ -156,3 +149,16 @@ func scheduleTask(s *eremeticScheduler, request types.Request) (string, error) {
...
 	return task.ID, nil
 }
+
+func updateStatusForTask(status *mesos.TaskStatus) {
+	id := status.TaskId.GetValue()
+	log.Debugf("TaskId [%s] status [%s]", id, status.State)
+	task, _ := database.ReadTask(id)
+	newStatus := types.Status{
+		Status: status.State.String(),
+		Time:   time.Now().Unix(),
```
Comment (keis) [inline handler/scheduler.go]: there is a `TaskStatus.GetTimeStamp` not sure if it's any better to use that.

File: handler/scheduler.go (hunk 1 above)
Comment (alde) [inline handler/scheduler.go]: To have it in the right order, felt more logical to prepend than to reverse when it's rendered

File: handler/scheduler.go (hunk 2 above)
Comment (alde) [inline handler/scheduler.go]: it probably is, will look into it

File: handler/scheduler.go (hunk 1 above)
Comment (keis) [inline handler/scheduler.go]: hmm ok. I guess it comes down to what the "right order" is :cat:

I do like the reverse chronological order in the presentation

**Comment by keis** [comment]
[LGTM](http://lgtm.in/i/Y8nqju1Tp)

**Pull request merged**